### PR TITLE
Add basic FAQ

### PR DESCRIPTION
**Pull request by volker-fr**
Really basic FAQ with some high level information. Requires a lot more work.

Related to #105

### CODE REVIEW

**Comment by volker-fr** [comment]
There is also some part in the repos main README, maybe a good idea to move it out from the main page? This PR is half draft.

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+For more advanced imports & formating, you can build your own templates.
+
+
+## Unanswered questions
+- Image imports
+    - Can the plugin import images from the Zotero annotation?
+    - What to check when image import fails
+- Templates import some PDFs with highlight colors, and others without color (all PDFs have been anotated with Zotero 6)
+- Does the Bibliography Style need to be set on the Zotero side, too? Or just being available.
```
Comment (mgmeyers) [inline docs/FAQ.md:35]: It just needs to be installed and does not need to be active!

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+The easiest way is to extract annotations and use the "Insert notes into current document". This requires that you export the annotations in Zotero first. Example import:
+
+<img src="https://raw.githubusercontent.com/mgmeyers/obsidian-zotero-integration/main/screenshots/demo.gif" alt="A short gif demonstraiting importing notes form Zotero into the current file">
+
+For more advanced imports & formating, you can build your own templates.
+
+
+## Unanswered questions
+- Image imports
+    - Can the plugin import images from the Zotero annotation?
```
Comment (mgmeyers) [inline docs/FAQ.md:32]: Yes, rectangle annotations are extracted and imported as images both in Zotero's native PDF reader as well as other PDF readers.

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    - What to check when image import fails
```
Comment (mgmeyers) [inline docs/FAQ.md:33]: Developer console and the image output path / base name:

Screen Shot 2022-10-09 at 9 28 33 AM

File: docs/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- Templates import some PDFs with highlight colors, and others without color (all PDFs have been anotated with Zotero 6)
```
Comment (mgmeyers) [inline docs/FAQ.md:34]: Can you share an example of what this looks like in Data Explorer?

**Review by mgmeyers — APPROVED** [review]
This is great, thanks @volker-fr ! I've added a few responses to your unanswered questions. Let me know if you want to address those before merging.

File: docs/FAQ.md (hunk 2 above)
Comment (mgmeyers) [reply docs/FAQ.md:32]: I just released a new version that fixed some issues with image annotations

File: docs/FAQ.md (hunk 4 above)
Comment (volker-fr) [reply docs/FAQ.md:34]: Ok, looks like this was a false-positive. I had the annotations in a separate note that wasn't colorized.

The entire PDF in Zotero has annotations, but the data object the Zotero Data Explorer sees has zero annotations. I can export the annotations into its own note in Zotero, though.

**Comment by volker-fr** [comment]
@mgmeyers  I did another update based on the information you provided and the one bug I just reported.

Please take another look and see whats missing/can be improved :)

**Pull request merged**